test/active_model_serializers/model_test.rb

```diff
@@ -18,5 +18,52 @@ def test_initialization_with_string_keys
 
       assert_equal model_instance.read_attribute_for_serialization(:key), value
     end
+
+    def test_attributes_can_be_read_for_serialization
```

Comment:
oh, I guess so.  I guess I haven't completely switch to activesupport form.  I mostly just wanted the stable interface vs. minitest